docs/modules/ROOT/pages/infra_services/data_plane_manager.adoc

```diff
@@ -178,157 +178,195 @@ anchor:Create_Port_NetworkConfiguration[]
-		"fast_path": true
-	}, {
-		"project_id": "3dda2801-d675-4688-a63f-dcda8d327f50",
-		"id": "f37810eb-7f83-45fa-a4d4-1b31e75399d0",
-		"name": "test_cni_port3",
-		"description": "",
-		"binding_host_ip": "10.213.43.187",
-		"vpc_id": "9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
-		"tenant_id": null,
-		"admin_state_up": true,
-		"mac_address": "86:ea:77:ad:52:56",
-		"veth_name": "veth0",
-		"device_id": null,
-		"device_owner": null,
-		"status": null,
-		"neighbor_host_ip": null,
-		"fixed_ips": [{
-			"ip_address": "192.168.2.3",
-			"subnet_id": "a87e0f87-a2d9-44ef-9194-9a62f178594f"
-		}],
-		"allowed_address_pairs": null,
-		"extra_dhcp_opts": null,
-		"security_groups": null,
-		"binding:host_id": "ephost_0",
-		"binding:profile": null,
-		"binding:vnic_type": null,
-		"network_ns": "/var/run/netns/test_netw_ns",
-		"dnsName": null,
-		"dnsAssignment": null,
-		"fast_path": true
-	}, {
-		"project_id": "3dda2801-d675-4688-a63f-dcda8d327f50",
-		"id": "f37810eb-7f83-45fa-a4d4-1b31e75399d6",
-		"name": "test_cni_port4",
-		"description": "",
-		"binding_host_ip": "10.213.43.188",
-		"vpc_id": "9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
-		"tenant_id": null,
-		"admin_state_up": true,
-		"mac_address": "86:ea:77:ad:52:57",
-		"veth_name": "veth0",
-		"device_id": null,
-		"device_owner": null,
-		"status": null,
-		"neighbor_host_ip": null,
-		"fixed_ips": [{
-			"ip_address": "192.168.2.4",
-			"subnet_id": "a87e0f87-a2d9-44ef-9194-9a62f178594f"
-		}],
-		"allowed_address_pairs": null,
-		"extra_dhcp_opts": null,
-		"security_groups": null,
-		"binding:host_id": "ephost_0",
-		"binding:profile": null,
-		"binding:vnic_type": null,
-		"network_ns": "/var/run/netns/test_netw_ns",
-		"dnsName": null,
-		"dnsAssignment": null,
-		"fast_path": true
-	}, {
-		"project_id": "3dda2801-d675-4688-a63f-dcda8d327f50",
-		"id": "f37810eb-7f83-45fa-a4d4-1b31e75399d7",
-		"name": "test_cni_port5",
-		"description": "",
-		"binding_host_ip": "10.213.43.188",
-		"vpc_id": "9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
-		"tenant_id": null,
-		"admin_state_up": true,
-		"mac_address": "86:ea:77:ad:52:58",
-		"veth_name": "veth0",
-		"device_id": null,
-		"device_owner": null,
-		"status": null,
-		"neighbor_host_ip": null,
-		"fixed_ips": [{
-			"ip_address": "192.168.2.5",
-			"subnet_id": "a87e0f87-a2d9-44ef-9194-9a62f178594f"
-		}	
-		],
-		"allowed_address_pairs": null,
-		"extra_dhcp_opts": null,
-		"security_groups": null,
-		"binding:host_id": "ephost_0",
-		"binding:profile": null,
-		"binding:vnic_type": null,
-		"network_ns": "/var/run/netns/test_netw_ns",
-		"dnsName": null,
-		"dnsAssignment": null,
-		"fast_path": true
-	}],
-	"security_groups_internal": [{}, {}]
+   "vpcs_internal":[
+      {
+         "project_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
+         "id":"9192a4d4-ffff-4ece-b3f0-8d36e3d88038",
+         "name":"test_vpc",
+         "description":"",
+         "cidr":"192.168.0.0/16"
+      }
+   ],
+   "subnets_internal":[
```

Comment:
This subnet_internal is aligned with our InternalSubnetEntity, at least the following fields:
- portIdsPerHost
- gatewayMacs
- dhcpIds

https://github.com/futurewei-cloud/alcor/blob/83f950b8cb1d699761439c75e829da3bb66991b5/web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalSubnetEntity.java#L20